moabb/datasets/bnci.py

```diff
@@ -286,26 +328,32 @@ def _load_data_004_2015(subject, path=None, force_update=False,
     url = '{u}004-2015/{s}.mat'.format(u=base_url, s=subjects[subject - 1])
     filename = data_path(url, path, force_update, update_path)[0]
 
-    ch_names = ['AFz', 'F7', 'F3', 'Fz', 'F4', 'F8', 'FC3', 'FCz',
-                'FC4', 'T3', 'C3', 'Cz', 'C4', 'T4', 'CP3', 'CPz',
-                'CP4', 'P7', 'P5', 'P3', 'P1', 'Pz', 'P2', 'P4',
-                'P6', 'P8', 'PO3', 'PO4', 'O1', 'O2']
+    ch_names = [
+        'AFz', 'F7', 'F3', 'Fz', 'F4', 'F8', 'FC3', 'FCz', 'FC4', 'T3', 'C3',
+        'Cz', 'C4', 'T4', 'CP3', 'CPz', 'CP4', 'P7', 'P5', 'P3', 'P1', 'Pz',
+        'P2', 'P4', 'P6', 'P8', 'PO3', 'PO4', 'O1', 'O2'
+    ]
     ch_types = ['eeg'] * 30
 
     return _convert_mi([filename], ch_names, ch_types)
 
 
 @verbose
-def _load_data_009_2015(subject, path=None, force_update=False,
-                        update_path=None, base_url=BBCI_URL,
+def _load_data_009_2015(subject,
+                        path=None,
+                        force_update=False,
+                        update_path=None,
+                        base_url=BBCI_URL,
                         verbose=None):
     """Load data for 009-2015 dataset."""
     if (subject < 1) or (subject > 21):
         raise ValueError("Subject must be between 1 and 21. Got %d." % subject)
 
-    subjects = ['fce', 'kw', 'faz', 'fcj', 'fcg', 'far', 'faw', 'fax', 'fcc',
-                'fcm', 'fas', 'fch', 'fcd', 'fca', 'fcb', 'fau', 'fci', 'fav',
-                'fat', 'fcl', 'fck']
+    subjects = [
+        'fce', 'kw', 'faz', 'fcj', 'fcg', 'far', 'faw', 'fax', 'fcc', 'fcm',
+        'fas', 'fch', 'fcd', 'fca', 'fcb', 'fau', 'fci', 'fav', 'fat', 'fcl',
+        'fck'
+    ]
```

Comment:
As of now subjects are defined as numbers...need to make arbitrary subject ids work with the analysis I believe

Reply:
input parameter `subject` is an integer, this is just a list to convert `subject` into its subject name from downloaded files. 
no need to change anything